rand_distr/src/utils.rs

```diff
@@ -33,9 +33,13 @@ pub trait Float: Copy + Sized + cmp::PartialOrd
     fn pi() -> Self;
     /// Support approximate representation of a f64 value
     fn from(x: f64) -> Self;
+    /// Support converting to an unsigned integer.
+    fn into_ui(self) -> Option<u64>;
```

Comment:
Should be `into_u64` (or probably just `to_u64` since this is a `Copy` type)

Reply:
Done.